### PR TITLE
Setup dotenv for environment variables

### PR DESCRIPTION
**Pull request by debelistic**
### What does this PR do?

- Setup dotenv for environment variables
- Add environment variables

### Description of Task to be completed?

- Install ```dotenv``` file
- Add environment variables

### How should this be manually tested?

- Create a file named ```.env``` in root folder and add environment variables
- Import ```config``` from ```dotenv``` to use in any file

### Any background context you want to provide?

- Ensure to ignore the ```.env``` file, to protect secretes

### What are the relevant pivotal tracker stories?

- #167765383

### Implementation Plan

- https://docs.google.com/document/d/1G2GsbzQEKcLkzbvqNjJVPT640QpDynrRVVyay7v-E0s/edit?usp=sharing

### Questions:

### CODE REVIEW

**Comment by voke1** [comment]
@debelistic Great work. I'd suggest you change `.envSample` to just `.env` as this is required in the root  directory.

**Comment by debelistic** [comment]
> @debelistic Great work. I'd suggest you change `.envSample` to just `.env` as this is required in the root directory.

The real `.env` file should be ignored by in gitiginore, so it doesn't show up on github. It contains secrets as well

**Comment by voke1** [comment]
Yes. `.env.` should be gitignored as the same environmental variables will be configured on Travis CI for tests. In this case, I see `.envSample` as being redundant. How do you intend to use `.envSample`?

**Comment by debelistic** [comment]
> Yes. `.env.` should be gitignored as the same environmental variables will be configured on Travis CI for tests. In this case, I see `.envSample` as being redundant. How do you intend to use `.envSample`?

The sample file is so you can provide the environment variables without their values. That would give whoever is cloning the repo an idea the environment variables they are meant to set

**Review by victor-nach — COMMENTED** [review]
I don't think dotenv should be a dev dependency, you may want to check that

**Comment by voke1** [comment]
> > Yes. `.env.` should be gitignored as the same environmental variables will be configured on Travis CI for tests. In this case, I see `.envSample` as being redundant. How do you intend to use `.envSample`?
> 
> The sample file is so you can provide the environment variables without their values. That would give whoever is cloning the repo an idea the environment variables they are meant to set

Okay. cool. I thought as much. This should be fine

**Review by voke1 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -26,5 +25,8 @@
     "passport-local": "^1.0.0",
     "request": "^2.87.0",
     "underscore": "^1.9.1"
+  },
+  "devDependencies": {
+    "dotenv": "^6.2.0"
```
Comment (anneKay) [inline package.json]: Why did you move 'dotenv' to devdependency?

Comment (debelistic) [reply package.json]: I felt since its only need in development environment, it should be devDependency. But I have moved it to dependencies regardless

**Comment by debelistic** [comment]
> I don't think dotenv should be a dev dependency, you may want to check that

I've updated it

**Review by ugwumadu116 — APPROVED** [review]
Nice one. Thanks for your passion

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "cSpell.words": [
+    "authorshaven"
+  ]
+}
```
Comment (anneKay) [inline .vscode/settings.json]: Please gitignore this file

Comment (debelistic) [reply .vscode/settings.json]: > Please gitignore this file

Done
